<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KYE-1000415093" name="a64c4287d540" tests="3" failures="1" timestamp="23 Aug 2018 08:25:27 GMT" time="100.85" errors="0">
  <testcase name="t001_login" time="22.242" classname="cn.crazy.appium.testcases.ZhihuZhihuTest"/>
  <testcase name="t002_search" time="10.649" classname="cn.crazy.appium.testcases.ZhihuZhihuTest"/>
  <testcase name="t003_guanzhu" time="13.824" classname="cn.crazy.appium.testcases.ZhihuZhihuTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at cn.crazy.appium.testng.Assertion.fail(Assertion.java:66)
	at cn.crazy.appium.testng.Assertion.assertEquals(Assertion.java:20)
	at cn.crazy.appium.testcases.ZhihuZhihuTest.t003_guanzhu(ZhihuZhihuTest.java:75)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- t003_guanzhu -->
</testsuite> <!-- a64c4287d540 -->
